.github/ISSUE_TEMPLATE/prepare_minor_release.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Minor release tracking issue
+title: Prepare for v<release-tag>
+---
+
+Please see this documentation for more details baout release process: https://github.com/metal3-io/metal3-docs/blob/main/processes/releasing.md
+
+
+**Note**:
+* The following is based on the v1.7 minor release. Modify according to the tracked minor release.
+
+## Tasks
+
+* [ ] Add/edit JJBs to acommodate new build jobs and PR jobs for new release branches. [Prior art](https://gerrit.nordix.org/c/infra/cicd/+/20875)
+* [ ] Add Prow jobs for new release branches in Project-infra repo [Prior art](https://github.com/metal3-io/project-infra/pull/697)
+* [ ] Prepare dev-env to acommodate new release related changes and configurations [Prior art](https://github.com/metal3-io/metal3-dev-env/pull/1381)
+* [ ] Update Metal3 book. [Prior art](https://github.com/metal3-io/metal3-docs/pull/413)
+* [ ] Release Ironic, also check new image is created in quay. [Prior art](https://github.com/metal3-io/ironic-image/tags)
+* [ ] Pin Ironic in BMO and release BMO, also check new image is created in quay. [Prior art](https://github.com/metal3-io/baremetal-operator/pull/1679)
+* [ ] Release IPAM (Branch out, add branch protection and required tests, also check new image is created in quay).
```

Comment:
Why were documenting IPAM/BMO/Ironic releasing in CAPM3 repo, shouldn't we have this tracking issue in more generic repo, like docs or dev-env or community?  Most of these steps do not related to CAPM3 after all.

Reply:
I am not sure what is the best place for this. If docs seems like a better place I can add it there. I added it here because the CAPM3 minor release is most of the time dependent on these releases.

Reply:
CAPM template should be about things done in CAPM3, similar to releasing.md is repo specific. Then in metal3-docs there is one that covers the common/extra steps. IMO these issue templates should be the same.